### PR TITLE
Changed the color, code cleanup

### PR DESCRIPTION
**Pull request by ScratchnSniff0**
Fixed the colors, remodeled some buttons, cleaned up some unnecessary code (various imports not used).  I need everybody's most recent work though to really remodel and unify the design/layout.

### CODE REVIEW

File: simulationfactory/src/util/Stylesheet.js [hunk 1]
```diff
@@ -1,55 +1,52 @@
...
-    },
-    blue: {
-      color: "blue",
+      padding: Theme.spacing(3),
     },
     defaultButton: {
       variant: "contained",
-      color: "primary",
       size: "medium"
     },
```
Comment (caydenreynolds) [inline simulationfactory/src/util/Stylesheet.js:49]: This class has no effect on the appearance of the buttons. We should probably remove it, and stop referencing it on all the buttons.
I'm like 90% sure I made this class, and forgot to delete it. That's my bad.

File: simulationfactory/src/util/theme.html [hunk 2]
```diff
@@ -0,0 +1,193 @@
+<div class="mdc-components">
```
Comment (caydenreynolds) [inline simulationfactory/src/util/theme.html:1]: What is this file? Do we need it? How do we access it from the website?

**Review by caydenreynolds — CHANGES_REQUESTED** [review]

File: simulationfactory/src/util/theme.html (hunk 2 above)
Comment (ScratchnSniff0) [reply simulationfactory/src/util/theme.html:1]: This was one of my attempts at importing a theme that I forgot to remove.  We definitely don't need it. I'll figure out how to remove it later today, or early tomorrow.

File: simulationfactory/src/util/Stylesheet.js (hunk 1 above)
Comment (ScratchnSniff0) [reply simulationfactory/src/util/Stylesheet.js:49]: Good to go, I will take a look at it either later this evening or tomorrow morning!

**Comment by ScratchnSniff0** [comment]
I removed that code, did a little more remodeling, got some things to line up better. Added the about page mockup, wasn't sure if anybody DIDN'T want to be in it so I left everybodys names out until we can ask them. The github link is just a placeholder to ycps main page.

**Pull request merged**